config/deploy/prod.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
-server 'sul-h2-prod.stanford.edu', user: 'h2', roles: %w[web app db cron]
+server 'xsul-h2-prod.stanford.edu', user: 'h2', roles: %w[web app db cron]
```

Comment:
Is making the host name invalid here serving a purpose?

Reply:
Oh, yes. I meant to describe with a comment. Fixing.